microsoft-edge/webview2/how-to/javascript.md

```diff
@@ -30,6 +30,24 @@ Use the following functions to begin embedding JavaScript in your WebView app.
 | [ExecuteScriptAsync](/dotnet/api/microsoft.web.webview2.wpf.webview2.executescriptasync) | Run JavaScript in a WebView control. For more information, navigate to the Get Started tutorial. |
 | [OnDocumentCreatedAsync](/microsoft-edge/webview2/reference/win32/icorewebview2#addscripttoexecuteondocumentcreated) | Runs when the Document Object Model (DOM) is created. |
 
+<!-- ====================================================================== -->
+## Scenario: ExecuteScript JSON Encoded Results
+
+Because the result of `ExecuteScriptAsync` is JSON encoded, if the result evaluating the JavaScript is a string you will receive a JSON encoded string and not the value of the string. For example, the following script results in a string with the following value including the quotes at the start and end, and the escaping slashes. 
+ * Script: "example \" \' \\ "; 
```

Comment:
```suggestion
 * Script: `example \" \' \\ ";`
```
not sure the desired result.  This suggestion simply wraps the entire string as-is, in backticks ( in Markdown, that means rendering a mid-sentence literal code string as-is).

Reply:
maybe done